Index: src/app/log_in/log_in.tsx
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/app/log_in/log_in.tsx b/src/app/log_in/log_in.tsx
new file mode 100644
--- /dev/null	(date 1716537960636)
+++ b/src/app/log_in/log_in.tsx	(date 1716537960636)
@@ -0,0 +1,1 @@
+
diff --git a/src/app/profile/profile.tsx b/src/app/profile/profile.tsx
new file mode 100644
diff --git a/src/app/leaderboard/leaderboard.tsx b/src/app/leaderboard/leaderboard.tsx
new file mode 100644
